packages/language-service/lib/documents.ts

```diff
@@ -60,13 +60,13 @@ export class SourceMapWithDocuments {
 	}
 
 	public * getSourcePositions(position: vscode.Position, filter: (data: CodeInformation) => boolean = () => true) {
-		for (const mapped of this.findPositions(position, filter, this.embeddedDocument, this.sourceDocument, 'generatedOffsets', 'sourceOffsets')) {
+		for (const mapped of this.findPositions(position, filter, this.embeddedDocument, this.sourceDocument, 'generatedOffsets')) {
```

Comment:
I've cleaned up an API a little bit - `'sourceOffsets'` is a duplciated information here